PyPDF2/_page.py

```diff
@@ -298,7 +298,7 @@ def createBlankPage(
         deprecate_with_replacement("createBlankPage", "create_blank_page")
         return PageObject.create_blank_page(pdf, width, height)
 
-    def rotate(self, angle: float) -> "PageObject":
+    def rotate(self, angle: int) -> "PageObject":

```

Comment:
You'll want to apply this change to the deprecated `rotate_clockwise`, `rotateClockwise`, and `rotateCounterClockwise` function signatures so that mypy is happy.